src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -4026,7 +4027,7 @@ public Response deaccessionDataset(@Context ContainerRequestContext crc, @PathPa
             return badRequest(BundleUtil.getStringFromBundle("datasets.api.deaccessionDataset.invalid.version.identifier.error", List.of(DS_VERSION_LATEST_PUBLISHED)));
         }
         return response(req -> {
-            DatasetVersion datasetVersion = getDatasetVersionOrDie(req, versionId, findDatasetOrDie(datasetId), uriInfo, headers, false);
+            DatasetVersion datasetVersion = getDatasetVersionOrDie(req, versionId, findDatasetOrDie(datasetId), uriInfo, headers, false, false);
```

Comment:
FWIW: This shouldn't ever need a checkPerms when deaccessioned false - because includeDeaccessioned is false. Might be clearer if this were just getDatasetVersionOrDie(req, versionId, findDatasetOrDie(datasetId), uriInfo, headers);

Reply:
Good catch! 🐟